crates/sui-indexer/migrations_v2/2023-08-19-044020_events/up.sql

```diff
@@ -18,7 +18,7 @@ CREATE TABLE events
     PRIMARY KEY(tx_sequence_number, event_sequence_number)
 );
 
-CREATE INDEX events_senders ON events USING GIN(senders);
-CREATE INDEX events_package_module ON events (package, module);
-CREATE INDEX events_event_type ON events (event_type);
+CREATE INDEX events_package ON events (package, tx_sequence_number, event_sequence_number);
+CREATE INDEX events_package_module ON events (package, module, tx_sequence_number, event_sequence_number);
```

Comment:
Covering indices are the most useful when they are used in order. So if we only have `events_package_module`, then it won't be used when the query is like `where package = A and tx_sequence_number > B and event_sequence_number > C`. But I didn't try it without `events_package` so i'm not sure if it will use Index Scan (with the primary keys) or Seq Scan